compiler-core/src/warning.rs

```diff
@@ -957,6 +957,8 @@ Your code will crash before reaching this point.",
                         PanicPosition::LastFunctionArgument =>
                             "This function call is unreachable because its last argument always panics. \
 Your code will crash before reaching this point.",
+                        PanicPosition::EchoExpression =>
+                            "This `echo` won't print anything because the expression it should be printing always panics.",
```

Comment:
This line seems like it might be long enough to break the formatting in this code